src/dune_pkg/resolved_package.ml

```diff
@@ -84,10 +84,15 @@ let scan_files_entries path =
          ])
 ;;
 
-let dune_package loc opam_file opam_package =
+let local_package ~command_source loc opam_file opam_package =
+  let dune_build =
+    match (command_source : Local_package.command_source) with
```

Comment:
I believe our style guide states that in such cases where the compiler doesn't know which constructors we are referring to, the constructors should be prefixed with the module instead of a type annotation. But my memory might be hazy.

Reply:
Quoting for the working on the Dune codebase:
```
Avoid qualifying constructors and record fields. Instead, add type annotations to the type being matched on or being constructed, e.g.,

Bad:
let foo = Command.Args.S []

Good:
let (foo : _ Command.Args.t) = S []

```

Reply:
Ah. I stand corrected!